### ISSUE TITLE
making the website responsive

### ISSUE THREAD

**Issue by shreyamaurya029** [Open-Source-Chandigarh/Cafe-Management#17]

**Comment by Kavyam172** [comment]
@shreyamaurya029 Please provide some description regarding the changes you want to make.

**Comment by shreyamaurya029** [comment]
yeah sure, responsive web design automatically adjusts for different-sized screens and viewports i.e., in android, tab , desktop

**Comment by Kavyam172** [comment]
@shreyamaurya029 I am assigning this issue to you. Go ahead with your idea.

**Comment by Kavyam172** [comment]
@shreyamaurya029 Are you working on this issue?

**Comment by Nitinshukla88** [comment]
Assign this issue to me.I'll try to find where website is unresponsive and correct it.
